resources/istio/templates/job-namespace-label.yaml

```diff
@@ -25,10 +25,23 @@ spec:
           - /bin/bash
           - -c
           - |
-            set -e
+            set +e
```

Comment:
I'd say it's against good bash practice. Every script should have "-e" at the top.
You can enclose individual statements, that are expected to cause errors, in the +e ... -e "guard", although I'd use that as a last resort technique. Try to use the command so that it doesn't fail or handle the failures at the place they occur.

Reply:
That was for debugging purposes during script development. Changed back to `-e`.